inc/Engine/Preload/Subscriber.php

```diff
@@ -375,4 +379,22 @@ public function preload_after_automatic_cache_purge( $deleted ) {
 			}
 		}
 	}
+
+	/**
+	 * Add the excluded uri from the preload to the filter.
+	 *
+	 * @param array $regexes regexes containing excluded uris.
+	 * @return array|false
+	 */
+	public function add_preload_excluded_uri( array $regexes ) {
+		$preload_excluded_uri = $this->options->get( 'preload_excluded_uri', false );
+
+		if ( ! $preload_excluded_uri ) {
+			return false;
+		}
+
+		$preload_excluded_uri = explode( "\n", $preload_excluded_uri );
```

Comment:
```suggestion
		$preload_excluded_uri = explode( "\n", $preload_excluded_uri );
```

The value of the option should be sanitized on saving to convert it to an array instead of a string, as we do for other textarea fields